addon.gypi

```diff
@@ -92,7 +92,7 @@
           '-luuid.lib',
           '-lodbc32.lib',
           '-lDelayImp.lib',
-          '-l"<(node_root_dir)/$(ConfigurationName)/<(node_lib_file)"'
+          '-l"<(node_root_dir)/<(target_arch)/<(node_lib_file)"'
```

Comment:
This change will break users on Windows who compile Node from source and use --nodedir to point at their source tree. When building from source, the node.lib file is written to either the `Release` or `Debug` directory depending on the build configuration used.